content/influxdb/v2.3/process-data/manage-tasks/create-task.md

```diff
@@ -119,29 +127,177 @@ option task = {
 
 ## Create a task using the InfluxDB API
 
-Send a request using the **POST** method to the `/api/v2/tasks` InfluxDB API endpoint.
-Provide the following in your API request:
+{{% oss-only %}}
+Use the [`/api/v2/tasks` InfluxDB API endpoint](/influxdb/v2.3/api/#operation/PostTasks) to create a task.
 
+{{< api-endpoint method="POST" endpoint="http://localhost:8086/api/v2/tasks/" >}}
+
+Provide the following in your API request:
 ##### Request headers
+
 - **Content-Type**: application/json
 - **Authorization**: Token *`INFLUX_API_TOKEN`*
 
 ##### Request body
+
 JSON object with the following fields:
 
-- **flux** : raw Flux task string
+- **flux** : raw Flux task string that contains [`options`](/flux/v0.x/spec/options/) and the query.
```

Comment:
"options" is pretty generic. There are a lot of options a script could contain. I believe they specifically look for the `task` option.